infrastructure/images/k6-action/Dockerfile

```diff
@@ -57,16 +57,18 @@ WORKDIR /jsonnet/vendor
 RUN jb init && \
     jb install github.com/jsonnet-libs/k8s-libsonnet/${K8S_LIBSONNET_VERSION}@main
 
+# Build Generator code
+COPY actions/generate-k6-manifests actions/generate-k6-manifests
+RUN cd actions/generate-k6-manifests && go build . && chmod +x generate-k6-manifests && mv generate-k6-manifests /usr/local/bin
 
+# Final image with the needed binaries and helper files
+FROM alpine:3.21.3@sha256:a8560b36e8b8210634f77d9f7f9efd7ffa463e380b75e2e74aff4511df3ef88c
+COPY --from=builder /usr/local/bin /usr/local/bin
```

Comment:
this would still include go and any other program in the `/usr/local/bin` folder. Do we need go in the final image?